src/Microsoft.VisualStudio.ProjectSystem.Managed/xlf/Resources.cs.xlf

```diff
@@ -96,6 +91,11 @@ Tento projekt se načetl pomocí nesprávného typu projektu, pravděpodobně v
         <target state="translated">Projekty</target>
         <note />
       </trans-unit>
+      <trans-unit id="Property_NoneValue">
+        <source>(None)</source>
+        <target state="new">(None)</target>
+        <note>Special value to show in the project property pages when a property has no value.</note>
+      </trans-unit>
```

Comment:
It would have been possible here to preserve the prior translations via replace-in-files. As done here, they will need to be re-translated.

Reply:
I did change the note to reflect its use in a more generic context (it was previously used on the NeutralLanguage property), so that triggered changes in the translations.

Reply:
Was able to save these translations when I fixed the merge conflicts this was having with main :)